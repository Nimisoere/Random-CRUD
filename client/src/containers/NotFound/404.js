import React from "react";
import { Container, Row, Col, Card, CardBody } from "reactstrap";

export const NotFound = () => (
  <Container>
    <Row className="text-center">
      <Col md={12}>
        <Card>
          <CardBody>
            <h1 className="font-weight-bold">404</h1>
            <p>Page not found</p>
          </CardBody>
        </Card>
      </Col>
    </Row>
  </Container>
);

export default NotFound;
