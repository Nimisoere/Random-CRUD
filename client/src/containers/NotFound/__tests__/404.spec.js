import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import NotFound from "../404";

Enzyme.configure({ adapter: new Adapter() });

function setup() {
  const enzymeWrapper = shallow(<NotFound />);
  return {
    enzymeWrapper
  };
}

describe("components", () => {
  describe("NotFound", () => {
    it("should match snapshot", () => {
      const { enzymeWrapper } = setup();
      expect(enzymeWrapper).toMatchSnapshot();
    });
  });
});
