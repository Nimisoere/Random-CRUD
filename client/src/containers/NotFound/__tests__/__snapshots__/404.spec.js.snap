// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components App should render 1`] = `ShallowWrapper {}`;

exports[`components NotFound should match snapshot 1`] = `ShallowWrapper {}`;

exports[`components NotFound should render 1`] = `ShallowWrapper {}`;
